### ISSUE TITLE
Bring Back "Making a Blog" Guide to Docs

### ISSUE THREAD

**Issue by westonganger** [locomotivecms/engine#1309]
The old v2.x Making a Blog Guide was extremely helpful to me as a newbie learning the concepts. I would love it if this was updated and included within the new docs somewhere.

http://doc-v2.locomotivecms.com/making-blog

**Comment by did** [comment]
I agree with you on that one. @jaredklewis did a wonderful job to write that tutorial.
We've to update it for v4.
